diff -up kid3-3.9.1/src/core/model/coretaggedfileiconprovider.h.omv~ kid3-3.9.1/src/core/model/coretaggedfileiconprovider.h
--- kid3-3.9.1/src/core/model/coretaggedfileiconprovider.h.omv~	2022-01-19 03:52:56.714595187 +0100
+++ kid3-3.9.1/src/core/model/coretaggedfileiconprovider.h	2022-01-19 03:53:05.152638896 +0100
@@ -27,6 +27,7 @@
 #pragma once
 
 #include "kid3api.h"
+#include <QByteArray>
 
 class QVariant;
 class QSize;
diff -up kid3-3.9.1/src/plugins/acoustidimport/ffmpegfingerprintdecoder.cpp.omv~ kid3-3.9.1/src/plugins/acoustidimport/ffmpegfingerprintdecoder.cpp
--- kid3-3.9.1/src/plugins/acoustidimport/ffmpegfingerprintdecoder.cpp.omv~	2022-01-19 03:53:45.016845019 +0100
+++ kid3-3.9.1/src/plugins/acoustidimport/ffmpegfingerprintdecoder.cpp	2022-01-19 03:54:03.604940928 +0100
@@ -232,7 +232,7 @@ private:
   friend class Format;
   friend class Converter;
   AVCodecContext* m_ptr;
-  AVCodec* m_impl;
+  const AVCodec* m_impl;
   AVFrame* m_frame;
   bool m_opened;
 };
